my-project/templates/product_category/_form.html.twig

```diff
@@ -10,4 +10,4 @@
 {{ form_start(form) }}
     {{ form_widget(form) }}
     <button class="btn">{{ button_label|default('Save') }}</button>
-{{ form_end(form) }}
```

Comment:
@mSobonVirtua do not commit such changes id not nessessary, better to rever the file before commit if inly such changes are made